mm2src/coins/lp_coins.rs

```diff
@@ -2461,9 +2467,10 @@ impl HttpStatusCode for WithdrawError {
             WithdrawError::HwError(_) => StatusCode::GONE,
             #[cfg(target_arch = "wasm32")]
             WithdrawError::BroadcastExpected(_) => StatusCode::BAD_REQUEST,
-            WithdrawError::Transport(_) | WithdrawError::InternalError(_) | WithdrawError::DbError(_) => {
-                StatusCode::INTERNAL_SERVER_ERROR
-            },
+            WithdrawError::Transport(_)
```

Comment:
`502 Bad Gateway` would be a better choice for converting `Transport` errors I think

Reply:
hmm, yes. according to context where we return `WithdrawError::Transport` it should be `Bad Gateway` status. 
changed